src/randoop/reflection/PackageVisibilityPredicate.java

```diff
@@ -0,0 +1,88 @@
+package randoop.reflection;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+
+/**
+ * A predicate that tests for visibility of a class, method, constructor, or
+ * field relative to a particular local package.
+ * For classes, tests if the class is either public or package private in
+ * the local package.
+ * Otherwise, implements a test for Java package accessibility rules, and
```

Comment:
Parts of this class use the term "accessibility", and parts of it use "visibility".  Please use only one or the other, or explain the distinction between the two terms.